requirements.in

```diff
@@ -0,0 +1,2 @@
+typing==3.7.4.3
+codecov==2.1.8
```

Comment:
We shouldn't pin version in requirements.in unless otherwise needed. It can have criteria for versions.

Reply:
BTW, codecov is a test dependency. It shouldn't be part of main requirements.in. I would suggest to create a separate test-requirements.{in,txt} & use it from cico scripts.